src/wrappers/themis/jsthemis/secure_session.hpp

```diff
@@ -31,6 +31,11 @@ class SecureSession : public Nan::ObjectWrap
 public:
     static void Init(v8::Local<v8::Object> exports);
 
+    bool isCreated()
+    {
+        return session_ != nullptr;
+    }
```

Comment:
I guess this method can be private.

(I know, no one *really cares* in this case, but good habits are important.)

Reply:
agree